### PR TITLE
DOC: Fix formatting issues in description of .c.src files

### PR DESCRIPTION
**Pull request by eric-wieser**
https://16017-908607-gh.circle-artifacts.com/0/doc/build/html/reference/distutils_guide.html#other-files

### CODE REVIEW

File: doc/DISTUTILS.rst.txt [hunk 1]
```diff
@@ -394,37 +394,37 @@ and ``/**end repeat**/`` lines, which may also be nested using
 consecutively numbered delimiting lines such as ``/**begin repeat1``
 and ``/**end repeat1**/``:
 
-1. "/\**begin repeat "on a line by itself marks the beginning of
-a segment that should be repeated.
+1. ``/**begin repeat ``on a line by itself marks the beginning of
```
Comment (eric-wieser) [inline doc/DISTUTILS.rst.txt:397]:
```suggestion
1. ``/**begin repeat`` on a line by itself marks the beginning of
```

**Comment by charris** [comment]
Hmm, the CircleCI failure should have been fixed by pinning pygments. But that was undone in #17332. @mattip Did you intend that?

**Comment by charris** [comment]
Thanks Eric.

**Comment by eric-wieser** [comment]
CircleCI does not build against the latest master - it's my fault CI fails here for not pulling first.

**Pull request merged**